core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2400,6 +2400,23 @@ private CarbonCommonConstants() {
    */
   public static final String CARBON_SI_SEGMENT_MERGE_DEFAULT = "false";
 
+  /**
+   * Until the threshold for complex filter is reached, row id will be set to the bitset in
+   * implicit filter during secondary index pruning
+   */
+  public static final String SI_COMPLEX_FILTER_THRESHOLD = "carbon.si.complex.filter.threshold";
+
+  /**
+   * Maximum value for complex filter threshold
+   */
+  public static final String SI_COMPLEX_FILTER_THRESHOLD_DEFAULT = "32000";
+
+  /**
+   * Property to decide if position id till row level or not
+   */
+  public static final String IS_TUPLE_ID_TILL_ROW_FOR_SI_COMPLEX =
```

Comment:
In the community, we concluded that no need of row level position reference. So, why this is required ?

Reply:
I think above 2 properties also not needed if it is not row level position reference

Reply:
CC: @kunal642 , @QiangCai

Reply:
cc: @kunal642 , @QiangCai , @ravipesala